etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
IMHO, the schema allows to update other PRs, You can go ahead and make changes. We can all review your changes, you dont need to be blocked for a sample at this stage of our development